frontend/packages/console-app/src/components/nodes/node-dashboard/StatusCard.tsx

```diff
@@ -19,6 +20,7 @@ const StatusCard: React.FC = () => {
       <DashboardCardBody isLoading={!obj}>
         <NodeHealth />
         <NodeAlerts />
+        <DashboardAlerts labelFilters={{ node: obj.metadata.name }} />
```

Comment:
nit: `labelSelector` might be a better name since it matches what the k8s API uses.

Reply:
@spadgett updated labelFilter to labelSelector